README.md

```
@@ -8,6 +8,41 @@ Plataforma de gestión de agentes IA, flujos, canales de mensajería y workspace
+
+### Si ya clonaste sin submódulos
+
+```bash
+git submodule update --init --recursive
+```
+
+### Qué contiene `vendor/agency-agents/`
+
+Repositorio externo [`msitarzewski/agency-agents`](https://github.com/msitarzewski/agency-agents) con templates de agentes por departamento:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)